test/sky-pages-route-generator.spec.js

```diff
@@ -104,4 +104,70 @@ describe('SKY UX Builder route generator', () => {
     expect(suppliedPattern).toEqual('my-custom-src/my-custom-pattern');
   });
 
+  it('should support guards with custom routesPattern', () => {
+    spyOn(glob, 'sync').and.callFake(() => ['my-custom-src/my-custom-route/index.html']);
+    spyOn(fs, 'readFileSync').and.returnValue('@Injectable() export class Guard {}');
+    spyOn(fs, 'existsSync').and.returnValue(true);
+
+    let routes = generator.getRoutes({
+      runtime: {
+        srcPath: 'my-custom-src/',
+        routesPattern: 'my-custom-pattern',
+      }
+    });
+
+    expect(routes.declarations).toContain(
+      `canActivate: [require(\'my-custom-src/my-custom-route/index.guard.ts\').Guard]`
+    );
+
+    expect(routes.declarations).toContain(
+      `canDeactivate: [require(\'my-custom-src/my-custom-route/index.guard.ts\').Guard]`
+    );
+
+    expect(routes.providers).toContain(
+      `require(\'my-custom-src/my-custom-route/index.guard.ts\').Guard`
+    );
+  });
+
+  it('should support default export guards', () => {
+    spyOn(glob, 'sync').and.callFake(() => ['my-custom-src/my-custom-route/index.html']);
+    spyOn(fs, 'readFileSync').and.returnValue('@Injectable() export default class Guard {}');
+    spyOn(fs, 'existsSync').and.returnValue(true);
+
+    let routes = generator.getRoutes({
+      runtime: {
+        srcPath: 'my-custom-src/',
+        routesPattern: 'my-custom-pattern',
+      }
+    });
+
+    expect(routes.declarations).toContain(
```

Comment:
Removing support for `export default` entirely now as part of refactoring to use `import` instead of `require`